modules/sdk-coin-bsc/test/unit/transactionBuilder/send.ts

```diff
@@ -63,16 +63,16 @@ describe('BSC Transfer Builder', () => {
       const A_sign_combine = MPC.signCombine({
         gShare: A_second_sign_convert.gShare!,
         signIndex: {
-          i: 1,
-          j: 2,
+          i: 2,
```

Comment:
This wasn't introduced in your rangeproof PR, but I believe these are backwards for all the tests. If you inspect them, they all have matching i and j